Library/Homebrew/dev-cmd/boneyard-formula-pr.rb

```diff
@@ -0,0 +1,129 @@
+# Creates a pull request to boneyard a formula.
+#
+# Usage: brew boneyard-formula-pr [options...] <formula-name>
+#
+# Options:
+#   --dry-run:  Print what would be done rather than doing it.
+
+require "formula"
+require "utils/json"
+require "fileutils"
+
+begin
+  require "json"
+rescue LoadError
+  puts "Homebrew does not provide Ruby dependencies; install with:"
+  puts "  gem install json"
+  odie "Dependency json is not installed."
+end
+
+module Homebrew
+  def boneyard_formula_pr
+    formula = ARGV.formulae.first
+    odie "No formula found!" unless formula
+
+    formula_path = formula.path
+    formula_file = "#{formula.name}.rb"
+    bottle_block = File.read(formula_path).include? "  bottle do"
+    if ARGV.dry_run?
+      puts "#{HOMEBREW_BREW_FILE} update"
+      puts "#{HOMEBREW_BREW_FILE} tap homebrew/boneyard"
+      puts "cd #{formula.tap.path}"
+      cd formula.tap.path
+      puts "cp #{formula_path.relative_path_from(formula.tap.path)} #{Tap.fetch("homebrew", "boneyard").formula_dir}"
+      puts "git rm #{formula_path.relative_path_from(formula.tap.path)}"
+      unless File.exist? formula.tap.path/"tap_migrations.json"
+        puts "Creating tap_migrations.json for #{formula.tap.name}"
+        puts "git add #{formula.tap.path}/tap_migrations.json"
+      end
+      puts "Loading tap_migrations.json"
+      puts "Adding #{formula.name} to tap_migrations.json"
+    else
+      safe_system HOMEBREW_BREW_FILE, "update"
+      safe_system HOMEBREW_BREW_FILE, "tap", "homebrew/boneyard"
```

Comment:
OK. Also, I just added `boneyard_tap = Tap.fetch("homebrew", "boneyard")` at the top and replaced all other instances.